Automattic/jetpack
# Search: Add E2E tests for Customberg ( the search customizer )

#### Changes proposed in this Pull Request:
The PR adds E2E tests for Customberg. It checks whether the theme, highlight color, product format and default sort would stick and reflected by the settings interface and the prview interface.

#### Jetpack product discussion
PcNPJE-uw-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
Setup and build: 
* `cd projects/plugins/jetpack/tests/e2e`
* `pnpm install`
* `pnpx jetpack build packages/search`
* `pnpx jetpack build plugins/jetpack`
* `pnpm env-start`
* `pnpm tunnel-on`
Run the Customberg E2E tests:
* Ensure test pass: `pnpm test-e2e -- ./specs/post-connection/search-configure.test.js`
* Ensure all E2E tests pass on GitHub

Note: you can add `--headed` at the end of the command to run Chromium in the foreground.
